app/assets/javascripts/components/case_listing/case_listing_controller.js

```diff
@@ -25,11 +27,18 @@ angular.module('QuepidApp')
       ctrl.submit     = submit;
 
       // we may get bound to different cases on moves, reset the state
-      $scope.$watch('thisCase', function() {
-        ctrl.thisCase            = $scope.thisCase;
-        ctrl.clickToEdit.currVal = ctrl.thisCase.caseName.slice(0);
-        ctrl.clickToEdit.clicked = false;
-      });
+      //$scope.$watch('thisCase', function() {
```

Comment:
@worleydl I ended up adding the caseRenamed watch, and it worked in one place and broke elsewhere..  so then I commented it out, and while updates take a ms longer to show, they do and i thought maybe this was safer...

Reply:
This is the only part of the PR that makes me nervous...why are we keeping the commented code around?

Reply:
Mostly cause I wanted you to see it!  I will delete now...